cli/package.json

```diff
@@ -14,6 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@differentialhq/core": "^3.16.0",
```

Comment:
Ideally we will try and use the `@differentialhq/core` version from the project if available.